sintia/sintia.py

```diff
@@ -261,6 +261,27 @@ def is_rate_limited(self, action: str, user_id: int) -> bool:
             return await message.channel.send(f'No results found for `{argument}`')
 
         return await message.channel.send(search_result["link"])
+    
+    @command_handler('gif')
+    async def google_image_search(self, message: discord.Message , argument: str) -> None:
+        if not argument:
+            return
+
+        google_config = get_config_section('search.google')
+        results = await self.http_get_request('https://www.googleapis.com/customsearch/v1?' + urlencode({
```

Comment:
I would prefer if you reuse the logic from `google_image_search` instead of copy-pasting the logic.